submissions/Vas-P/popup/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./css/style.css" type="text/css">
+	<title>No js popup</title>
+</head>
+<body>
+	<nav>
+		<ul class="nav">
+			<li><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO" class="no" target="_blank">Don't click!</a></li>
+			<li><a href="#" class="images">Images</a></li>
```

Comment:
What is means the element class "no"?
Use more informative names of the class. Also, you can use [BEM](https://ru.bem.info/methodology/quick-start/) methodology.